plc4go/internal/opcua/EncryptionHandler.go

```diff
@@ -131,9 +131,9 @@ func (h *EncryptionHandler) decodeMessage(ctx context.Context, pdu readWriteMode
 		var message []byte
 		switch pduMessage := pdu.GetMessage().(type) {
 		case readWriteModel.OpcuaOpenResponseExactly:
-			message = pduMessage.GetMessage()
+			message = pduMessage.(readWriteModel.BinaryPayloadExactly).GetPayload()
 		case readWriteModel.OpcuaMessageResponseExactly:
-			message = pduMessage.GetMessage()
+			message = pduMessage.(readWriteModel.BinaryPayloadExactly).GetPayload()
```

Comment:
is it ensured that this is always `readWriteModel.BinaryPayloadExactly`?. Otherwise this part will panic